### ISSUE TITLE
Issue with 6.3.0

### ISSUE THREAD

**Issue by pmarangoni** [twilio/twilio-csharp#678]
Running a site with .NET Framework 4.8.
Twilio fails with:

```
Method not found: 'System.String[] System.String.Split(Char, System.StringSplitOptions)'
Method Name: System.Uri buildUri()
Class Defining Member: Twilio.Http.Request
Stack Trace
   at Twilio.Http.Request.buildUri()
   at Twilio.Http.Request.ConstructUrl()
   at Twilio.Clients.TwilioRestClient.Request(Request request)
   at Twilio.Rest.Api.V2010.Account.MessageResource.Create(CreateMessageOptions options, ITwilioRestClient client)
   at Twilio.Rest.Api.V2010.Account.MessageResource.Create(PhoneNumber to, String pathAccountSid, PhoneNumber from, String messagingServiceSid, String body, List`1 mediaUrl, Uri statusCallback, String applicationSid, Nullable`1 maxPrice, Nullable`1 provideFeedback, Nullable`1 attempt, Nullable`1 validityPeriod, Nullable`1 forceDelivery, ContentRetentionEnum contentRetention, AddressRetentionEnum addressRetention, Nullable`1 smartEncoded, List`1 persistentAction, Nullable`1 shortenUrls, ScheduleTypeEnum scheduleType, Nullable`1 sendAt, Nullable`1 sendAsMms, String contentSid, String contentVariables, ITwilioRestClient client)
```

I have reverted to 6.2.5 and all is well.

**Comment by keltex** [comment]
I confirm that I'm seeing the exact same problem. Reverted to 6.2.5 and works fine. Also .NET Framework 4.8

`System.MissingMethodException: Method not found: 'System.String[] System.String.Split(Char, System.StringSplitOptions)'.

at Twilio.Http.Request.buildUri
at Twilio.Http.Request.ConstructUrl
at Twilio.Clients.TwilioRestClient.<RequestAsync>d__25.MoveNext
at Twilio.Rest.Api.V2010.Account.MessageResource.<CreateAsync>d__8.MoveNext
at Twilio.Rest.Api.V2010.Account.MessageResource.<CreateAsync>d__10.MoveNext`

**Comment by sbansla** [comment]
twilio-csharp is used to be build using dotnet-version: '3.1.x' ([link](https://github.com/twilio/twilio-csharp/blob/main/.github/workflows/test-and-deploy.yml#L27))
This time(6.3.0) it was build using .net 6 because of which certain dependencies are not installed.
This will be fixed in upcoming release.

**Comment by maginfo** [comment]
we had to revert to 6.2.5 also, after receiving the error above in 6.3

**Comment by sbansla** [comment]
Hi @maginfo,
We have releases 6.4.0 version, Can you please test with this version.

**Comment by danFbach** [comment]
Confirm 6.4.0 fixes this issue, running new framework 4.8
